lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -133,6 +133,8 @@
         group_desc: other example EC2 group
 '''
 
+import ipaddress
```

Comment:
@ryansb checked it out and boto doesn't do cidr_ip validation. So this validation is done on AWS backend.